spec/models/evidence/forms/evidence_spec.rb

```diff
@@ -60,17 +60,52 @@
     end
   end
 
-  describe '#save' do
-    before do
-      allow(subject).to receive(:valid?).and_return(true)
-      allow(subject).to receive(:persist!)
-      allow(EvidenceCheck).to receive(:find)
-      allow_any_instance_of(EvidenceCheck).to receive(:update)
-      allow_any_instance_of(Reason).to receive(:save)
+  describe '#save', focus: true do
```

Comment:
I've changed the testing approach to this method, even though I know it'll create a discussion around it. So I'd like to explain:

Because it's now setting outcome of the whole evidence flow, when the evidence is incorrect, I feel that this needs to be tested solid and making sure the right fields are updated on the right columns. 
So it's a bit slower than it was, but I think it's more robust. If we refactored the form to accept the `EvidenceCheck` in the constructor, we wouldn't have to store the evidence check in the database in the test. I think it would be cleaner to test.